apps/Standalone/src/designer/app/SettingsSections/sourceSettings.tsx

```diff
@@ -7,7 +7,10 @@ import { ChoiceGroup, IconButton } from '@fluentui/react';
 import { useEffect, useMemo } from 'react';
 import { useDispatch } from 'react-redux';
 
-const SourceSettings = () => {
+const SourceSettings = ({
```

Comment:
NIT: can we have the `showEnvironment` and `showHistory` as part of the redux store? Instead of sending them in the props? 
So we can use them  as folllowing? 
`const showEnvironment = useShowEnvironment();`

Reply:
Why, having them props is also fine, they dont change after first render and such props need not be in store... also this is just apps for test page

Reply:
ok, sounds good then. I was just suggesting it as it seems that that's the pattern we have been following.